src/selfplay/game.cc

```diff
@@ -89,14 +89,14 @@ void SelfPlayGame::Play(int white_threads, int black_threads, bool training,
           std::chrono::steady_clock::now() +
           std::chrono::milliseconds(options_[idx].search_limits.movetime);
     }
+
     {
       std::lock_guard<std::mutex> lock(mutex_);
       if (abort_) break;
       search_ = std::make_unique<Search>(
           *tree_[idx], options_[idx].network, options_[idx].best_move_callback,
           options_[idx].info_callback, options_[idx].search_limits,
-          *options_[idx].uci_options, options_[idx].cache, nullptr);
-      // TODO: add Syzygy option for selfplay.
+          *options_[idx].uci_options, options_[idx].cache, syzygy_tb);
```

Comment:
Maybe make it a `shared_ptr` in `options_` like other fields, to allow e.g. having tablebase vs no-tablebase training?

Reply:
I don't really see why we would want different tablebase setttings for player1 and player2 during training for one client?

In this PR, using TB is already optional. 
That is, some clients who have 4-5 piece TB installed can use it, most clients won't have it installed initially and all those games come together at the server for aggregation into a net.

In other words: what would one client training with and without help? Or is this just for generation of nets on a local system?

Reply:
@mooskagh rebase to master was quite difficult due to the number of changes and the time it takes to get these kind of suggestions in. Is there any way I can speed up or make merging easier?